git-bot/__main__.py

```diff
@@ -21,17 +22,22 @@ async def pr_opened_event(event, gh, *args, **kwargs):
     owner = event.data["repository"]["owner"]["login"]
     repo = event.data["repository"]["name"]
     comments_url = event.data["pull_request"]["comments_url"]
-    
+    import pickle
+    with open('filename.pickle', 'wb') as handle:
+        pickle.dump(event, handle, protocol=pickle.HIGHEST_PROTOCOL)
```

Comment:
Here’s a suggested change for Insecure use of pickle for serialization:

```suggestion
import json
with open('filename.json', 'w') as handle:
    json.dump(event, handle)
```